actool/manifest.go

```diff
@@ -83,6 +90,35 @@ func getIsolatorStr(name, value string) string {
                 }`, name, value)
 }
 
+func isolatorStrFromString(is string) (string, error) {
+	is = "name=" + is
+	v, err := url.ParseQuery(strings.Replace(is, ",", "&", -1))
+	if err != nil {
+		return "", err
+	}
+
+	var name string
+	var values []string
+
+	for key, val := range v {
+		if len(val) > 1 {
+			return "", fmt.Errorf("label %s with multiple values %q", key, val)
```

Comment:
@jonboulle I think `isolator` is not right here, because the key can be `name`, `request`, `limit`. Similar cases can be found: https://github.com/appc/spec/blob/master/schema/types/volume.go#L93